Docs: Add 0.8.0 to changelog

jaredbeck opened the pull request:
[ci skip]

radar commented on CHANGELOG.md:
```diff
@@ -2,6 +2,10 @@
 
+# 0.8.0
+
+See https://github.com/svenfuchs/i18n/releases/tag/0.8.0.beta1
```
I've just added https://github.com/svenfuchs/i18n/releases/tag/v0.8.0 to the releases list. Could you please update the link here to that one?

jaredbeck commented:
> I've just added https://github.com/svenfuchs/i18n/releases/tag/v0.8.0 to the releases list. Could you please update the link here to that one?

Updated in https://github.com/svenfuchs/i18n/pull/361

Closing.
